### PR TITLE
CO-149: Backport changes from 1.0.x branch

### PR DESCRIPTION
**Pull request by carlpulley**
- [x] Ensure all relevant changes from v1.0.5 of 1.x branch are merged
- [x] Merge in scalafmt.conf file from Akka HTTP
- [x] Updated README with need to use AshScriptPlugin
---
- [x] Ready for review

### CODE REVIEW

**Review by ferhtaydn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -294,19 +298,6 @@ being maintained and conflicting with [wartremover](https://github.com/wartremov
 
 No special tasks are enabled for this plugin.
 
-# Releasing of the `sbt-cake` Plugin to the Sonatype Repository
```
Comment (carlpulley) [inline README.md:297]: This text already exists at the end of the file

File: .scalafmt.conf [hunk 2]
```diff
@@ -1,20 +1,27 @@
-maxColumn = 80
-
-style = IntelliJ
-danglingParentheses = true
-indentOperator = spray
+align = none
```
Comment (carlpulley) [inline .scalafmt.conf]: This is the scalafmt.conf file we are currently planning to use on all projects

Comment (amirkarimi) [reply .scalafmt.conf]: Probably after this bug is fixed: https://github.com/scalameta/scalafmt/issues/994?

Comment (carlpulley) [reply .scalafmt.conf]: Thanks for the reminder, I'll add in a comment to the Jira ticket tracking this. Other than that, I'll leave this option enabled as it is only currently impacting Scala.js projects.

**Pull request merged**